PowerEditor/installer/nativeLang/french.xml

```diff
@@ -197,17 +197,17 @@
                     <Item id="43030" name="Appliquer le style 5"/>
                     <Item id="43031" name="Dégager le style 5"/>
                     <Item id="43032" name="Dégager tous les styles"/>
-                    <Item id="43033" name="Style 1"/>
-                    <Item id="43034" name="Style 2"/>
-                    <Item id="43035" name="Style 3"/>
-                    <Item id="43036" name="Style 4"/>
-                    <Item id="43037" name="Style 5"/>
+                    <Item id="43033" name="Recherche le Style 1 suivant"/>
+                    <Item id="43034" name="Recherche le Style 2 suivant"/>
+                    <Item id="43035" name="Recherche le Style 3 suivant"/>
+                    <Item id="43036" name="Recherche le Style 4 suivant"/>
+                    <Item id="43037" name="Recherche le Style 5 suivant"/>
                     <Item id="43038" name="Style de recherche"/>
-                    <Item id="43039" name="Style 1"/>
-                    <Item id="43040" name="Style 2"/>
-                    <Item id="43041" name="Style 3"/>
-                    <Item id="43042" name="Style 4"/>
-                    <Item id="43043" name="Style 5"/>
+                    <Item id="43039" name="Rechercher le Style 1 précédent"/>
+                    <Item id="43040" name="Rechercher le Style 2 précédent"/>
+                    <Item id="43041" name="Rechercher le Style 3 précédent"/>
+                    <Item id="43042" name="Rechercher le Style 4 précédent"/>
+                    <Item id="43043" name="Rechercher le Style 5 précédent"/>
```

Comment:
According to my GitHub diff-view you added the conjugation `Recherche` (**w/o** trailing **r**) for id range 43033 to 43037. For id range 43039 to 43043 you added `Rechercher` (**w/** trailing **r** i.e. the verb).
Yep, I know, I'm nit-picking ;-)